pcrmc/config.py

```diff
@@ -37,7 +37,7 @@ def _init_config_file() -> int:
 
 def _create_database(db_path: str) -> int:
     config_parser = configparser.ConfigParser()
-    config_parser["General"] = {"database": db_path}
+    config_parser["General"] = {"database": db_path, "RunningID": 0}
```

Comment:
Just a small thing, but we should specify in the naming whether the id is the current last id or the next id. In this case it would be the next id, so maybe name it "NextID" or something like that.

Reply:
done